src/actions/actionToggleZenMode.tsx

```diff
@@ -1,20 +1,21 @@
 import { CODES, KEYS } from "../keys";
 import { register } from "./register";
 
-export const actionToggleZenMode = register({
-  name: "zenMode",
-  perform(elements, appState) {
-    this.checked = !this.checked;
-    return {
-      appState: {
-        ...appState,
-        zenModeEnabled: this.checked,
-      },
-      commitToHistory: false,
-    };
-  },
-  checked: false,
-  contextItemLabel: "buttons.zenMode",
-  // Wrong event code
-  keyTest: (event) => event[KEYS.CTRL_OR_CMD] && event.code === CODES.QUOTE,
-});
+export const actionToggleZenMode = (checked: boolean = false) =>
+  register({
+    name: "zenMode",
+    perform(elements, appState) {
+      this.checked = !this.checked;
+      return {
+        appState: {
+          ...appState,
+          zenModeEnabled: this.checked,
+        },
+        commitToHistory: false,
+      };
+    },
+    checked,
+    contextItemLabel: "buttons.zenMode",
+    // Wrong event code
```

Comment:
and this.. :) can we remove it or put the _right_ code?

Reply:
no idea!  but fixed and it didn't get caught initially since it wasn't getting used :D